api/src/services/policies.ts

```diff
@@ -33,6 +84,12 @@ export class PoliciesService extends ItemsService<Policy> {
 	}
 
 	override async deleteMany(keys: PrimaryKey[], opts: MutationOptions = {}): Promise<PrimaryKey[]> {
+		if (opts.onRequireUserIntegrityCheck) {
+			opts.onRequireUserIntegrityCheck(UserIntegrityCheckFlag.All);
+		} else {
+			opts.userIntegrityCheckFlags = UserIntegrityCheckFlag.All;
```

Comment:
Same question as in https://github.com/directus/directus/pull/22737/files#r1639939752

Reply:
Same answer. 😆